README.md

```diff
@@ -97,61 +122,41 @@ cargo install cargo-c
 cargo cinstall --release
 ```
 
-## Compressing video
-
-Input videos must be in y4m format. The monochrome color format is not supported.
+## Usage
+### Compressing video
+Input videos must be in [y4m format](https://wiki.multimedia.cx/index.php/YUV4MPEG2). The monochrome color format is not supported.
 
 ```sh
 cargo run --release --bin rav1e -- input.y4m -o output.ivf
 ```
 
-## Decompressing video
+_(Find a y4m-file for testing at [`tests/small_input.y4m`](tests/small_input.y4m) or at http://ultravideo.cs.tut.fi/#testsequences)_
 
+### Decompressing video
 Encoder output should be compatible with any AV1 decoder compliant with the v1.0.0 specification. You can build compatible aomdec using the following:
 
 ```sh
-mkdir aom_test
-cd aom_test
+mkdir aom_test && cd aom_test
 cmake /path/to/aom -DAOM_TARGET_CPU=generic -DCONFIG_AV1_ENCODER=0 -DENABLE_TESTS=0 -DENABLE_DOCS=0 -DCONFIG_LOWBITDEPTH=1
 make -j8
 ./aomdec ../output.ivf -o output.y4m
 ```
 
-## Configuring
+### Configuring
+rav1e has several optional features that can be enabled by passing `--features` to cargo. Passing `--all-features` is discouraged.
 
-rav1e has several optional features that can be enabled by passing --features to cargo test. Passing --all-features is discouraged.
+#### Features
+Find a full list in feature-table in [`Cargo.toml`](Cargo.toml)
 
-* asm - enabled by default. When enabled, assembly is built for the platforms supporting it.
+* `asm` - enabled by default. When enabled, assembly is built for the platforms supporting it.
   * It requires `nasm` on `x86_64`.
   * It requires `gas` on `aarch64`.
```

Comment:
While at it might be good to mention that the clang assembler is also supported.

Reply:
Can you maybe add the instructions how this works?

Reply:
Addressed in https://github.com/xiph/rav1e/pull/2365/commits/b3de4e60a5c5562d5a9b8ecd2da8d3f0ad87b271